src/umpire/resource/CMakeLists.txt

```diff
@@ -17,11 +19,14 @@ set (umpire_resource_headers
 )
 
 set (umpire_resource_sources
```

Comment:
I'm not sure how this conversation got marked as resolved, but this entire `set` operation needs to be replaced with:
```
set (umpire_resource_sources
  FileMemoryResource.cpp
  FileMemoryResourceFactory.cpp
  HostResourceFactory.cpp
  MemoryResource.cpp
  MemoryResourceRegistry.cpp
  NullMemoryResource.cpp
  NullMemoryResourceFactory.cpp
)
```